release-calendar/src/client/components/Calendar.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import FullCalendar from '@fullcalendar/react';
+import dayGridPlugin from '@fullcalendar/daygrid';
+import timeGridPlugin from '@fullcalendar/timegrid';
+//import interactionPlugin from '@fullcalendar/interaction';
+
+import '../stylesheets/calendarStyle.scss';
+import {EventSourceInput} from '@fullcalendar/core/structs/event-source';
+
+export interface CalendarProps {
+  events: EventSourceInput[];
+}
+export interface CalendarState {
+  calendarWeekends: boolean;
+}
+
+export class Calendar extends React.Component<CalendarProps, CalendarState> {
+  constructor(props: Readonly<CalendarProps>) {
+    super(props);
+    this.state = {
+      calendarWeekends: true,
```

Comment:
Curious why you added this toggle weekend bit? Was it part of the mock up?

Reply:
It wasn't part of my mock up. It was something I hadn't considered. I added it mainly now because  the process of adding it was a good way to learn and interact with fullCalendar's documentation. Plus, I thought it couldnt hurt. If you think I should remove it, I'm happy to.

Reply:
Leaving it in is fine if you want to keep it, but I would style and place it to look similar to the other calendar buttons.

Reply:
I think weekends should always be displayed and no toggle available - in some countries the weekend is different days (e.g. in Israel the weekend is Friday and Saturday, not Sunday; in many Muslim countries the weekend is just Saturday...) - there's no strong reason to hide the weekends :)